auditlog/models/rule.py

```diff
@@ -343,6 +358,16 @@ def read(self, fields=None, load="_classic_read", **kwargs):
                 return result
             self = self.with_context(auditlog_disabled=True)
             rule_model = self.env["auditlog.rule"]
+            rule = rule_model.sudo().search([("model_id.model", "=", self._name)])
```

Comment:
could we benefit from mem cache here instead of querying everytime for these rules and fields?